### DISCUSSION TITLE
Controller file not found!

### DISCUSSION THREAD

**Discussion by upadesh** [givanz/Vvveb#76] [General]
When I try to access "Custom Posts" or "Custom Products" it says " Page not found! Error 404 Controller file not found! " . I double checked on the \admin\controller\settings\  folder and both the posts.php and products.php were missing.
Please help me where I can find those files because it is not even in the github folder. Thank you.

**Comment by givanz** [comment]
Sorry, user defined custom posts and products feature was not finished, they could be added only through plugins.

This feature is now added in the latest update https://github.com/givanz/Vvveb/commit/4933dd1aa2aaa5d04ed17307f6db5fa1a14d6554

https://demo.vvveb.com/admin/index.php?module=settings/post-types

You can redownload latest.zip and overwrite files on your server.

In next updates more options will be added like disable comments for custom posts or disable options, attributes etc for custom products.
